EIPS/eip-5851.md

```diff
@@ -0,0 +1,257 @@
+
+## Specification
+
+The keywords "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119 and RFC 8174.
+
+### Metadata Standard
+
+We will be storing the following structures to store the claims:
+
+### 1. Metadata information
+
+We will store the metadata corresponding ot the each claim requirement, the structure is similar to the [EIP-3475](./eip-3475.md):
+
+```solidity
+    /** Metadata
+    * 
+    * @param title defines the name of the claim field
+    * @param _type is the nature of the data (bool,string,address,bytes,..)
+    * @param description additional information about claim details.
+     */
+    struct Metadata {
+        string title;
+        string _type;
+        string description;
+    }
+```
+
+
+### 2. Values Information
+
+this structure will be used to define the actual claim information, based on the description of the Metadata structure, it can represent any data type represented by the solidity(or developers can define the generic type). 
+
+```solidity
+    /** Values
+    * 
+    * @dev Values here can be read and wrote by smart contract and front-end, cited from [EIP-3475](./eip-3475.md) 
+    */   
+    struct Values { 
+        string stringValue;
+        uint uintValue;
+        address addressValue;
+        bool boolValue;
+    }
+```
+
+### 3. Claim structure
+
+This will store the content about the claims that user wants to convey (ie age >= 18, jurisdiction not from the blacklisted countries, etc). these will be stored as the combination of one or many claim structure detail attributes as defined below: 
+
+```solidity
+    /** Claims
+    * 
+    * Claims structure consist of the conditions and value that holder claims to associate and verifier has to validate them.
+    * @notice the below given parameters are for reference purposes only, developers can optimize the fields that are needed to be represented on-chain by using schemes like TLV, encoding into base64 etc.
+    * @dev structure that DeFines the parameters for specific claims of the SBT certificate
+    * @notice this structure is used for the verification process, it contains the metadata, logic and expectation
+    * @logic given here MUST be one of ("⊄", "⊂", "<", "<=", "==", "!=", ">=",">")
```

Comment:
If these are the only permitted values, would it make more sense to use an enumeration or an integer?

---

Regardless of the above, the operations these symbols represent should be defined in this EIP.

Reply:
for now, I have defined some of the examples with the table giving the example. but given that there are numerous operations in the set operations, maintaining them in the standard will be redundant, and rather than that we can simply tell developers to follow standard ASCII operations in order to define the function logic.